providers/asrockrack/helpers.go

```diff
@@ -578,7 +578,7 @@ func (a *ASRockRack) httpsLogin(ctx context.Context) error {
 func (a *ASRockRack) httpsLogout(ctx context.Context) error {
 	_, statusCode, err := a.queryHTTPS(ctx, "api/session", "DELETE", nil, nil, 0)
 	if err != nil {
-		return fmt.Errorf("Error logging out: " + err.Error())
+		return errors.Wrap(err, "logging out")
```

Comment:
`github.com/pkg/errors` is actually archived. We should eventually remove its use entirely. You might think about using stdlib wrapping functionality for this. For example: `fmt.Errorf("logging out: %w", err)`

Reply:
I agree.   If it's not a big lift, I'd say let's do it here.   If it is, I'd like to see that in a follow-on PR.

Reply:
I’m definitely on-board with this. I’ll revise tomorrow.

Reply:
I did this for the code I changed, but left the existing usage.